src/modules/auth/googleauth/google.service.ts

```diff
@@ -0,0 +1,34 @@
+import { PassportStrategy } from '@nestjs/passport';
+import { Strategy, VerifyCallback } from 'passport-google-oauth20';
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class GoogleStrategy extends PassportStrategy(Strategy, 'google') {
+  constructor() {
+    super({
+      clientID: `${process.env.GOOGLE_CLIENT_ID}`,
+      clientSecret: `${process.env.GOOGLE_CLIENT_SECRET}`,
+      callbackURL: `http://${process.env.HOST}:${process.env.PORT}/auth/google/redirect`,
+      scope: ['email', 'profile'],
+    });
+    this.validate = this.validate.bind(this);
+  }
+
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
its strange create rule and then disable it everywhere

Reply:
fixed